lms/djangoapps/instructor/access.py

```diff
@@ -45,7 +49,7 @@ def allow_access(course, user, level):
     """
     Allow user access to course modification.
 
-    `level` is one of ['instructor', 'staff', 'beta']
+    `level` is one of ['instructor', 'staff', 'beta', 'ccx_coach']
```

Comment:
Good question... appears I was a bit over-zealous in making the various access levels consistent.